src/html/to_html_tree.ml

```diff
@@ -811,12 +811,26 @@ struct
           }
 
       | `Nested_article item ->
-        let html, subpages = state.render_nested_article item in
+        let html, maybe_docs, subpages = state.render_nested_article item in
         let html, maybe_id = add_anchor state.item_to_id item html in
         let a = add_spec state.item_to_spec item maybe_id in
+        let html =
+          match maybe_docs with
+          | [] -> Html.article ~a html
+          | docs ->
+            let docs = Documentation.first_to_html docs in
+            let docs = (docs :> (Html_types.div_content Html.elt) list) in
+            let adoc =
+              [ Html.a_class ["doc"]
+              ; Html.a_style "padding: 0.75em; padding_bottom: 0;" ]
```

Comment:
The style should be added to the CSS file. Also ideally we should reuse existing rules (see my comment about `dl` tags).